cypress/support/cypress-support/src/main.js

```diff
@@ -89,15 +89,6 @@ export default function (module) {
     Cypress.env(CONSTANTS.MESSAGE_QUEUE, messageQueue);
     UTILS.parseExceptionList();
     cy.getModuleReqIdJson();
-    if (UTILS.getEnvVariable(CONSTANTS.PERFORMANCE_METRICS) == true) {
-      cy.startOrStopPerformanceService(CONSTANTS.INITIATED).then((response) => {
-        if (response) {
-          Cypress.env(CONSTANTS.IS_PERFORMANCE_METRICS_ENABLED, true);
-        }
-      });
-    } else {
-      cy.log(CONSTANTS.PERFORMANCE_METRICS_NOT_ACTIVE);
-    }
```

Comment:
why did you remove this code ? This function is to start/stop metrics collection and that should reside in before operation

Reply:
reverted the change